docs/integrations/builtin/credentials/google/index.md

```diff
@@ -18,6 +18,16 @@ This section contains:
 
 There are two authentication methods available for Google services nodes, [OAuth2](https://developers.google.com/identity/protocols/oauth2){:target=_blank .external-link} and [Service Account](https://cloud.google.com/iam/docs/understanding-service-accounts){:target=_blank .external-link}. n8n recommends using OAuth. It's more widely available, and easier to set up. Refer to the [Google documentation: Understanding service accounts](https://cloud.google.com/iam/docs/understanding-service-accounts){:target=_blank .external-link} for guidance on when you need service account.
 
+/// note | Note for n8n Cloud users
```

Comment:
You could use the snippet here as well.

Reply:
The snippet I used for the other two pages includes the 'skip the instructions explained in this document' text. I could remove this and use the snippet for all three pages. Would that be better?

Reply:
Yes, as that way if the list changes we can easily make the update

Reply:
Sounds good!